template/template.py

```diff
@@ -0,0 +1,121 @@
+                xpaint \
+                tint2 \
+                galculator \
+                pcmanfm \
+                software-properties-common \
+                apt-transport-https \
+                libgtk-3-bin",
+        ]
+    )
+    .pip_install("numpy")
+    # Setup NoVNC and websockify
+    .run_cmd(
+        [
+            "git clone --branch e2b-desktop https://github.com/e2b-dev/noVNC.git /opt/noVNC",
+            "ln -s /opt/noVNC/vnc.html /opt/noVNC/index.html",
+            "git clone --branch v0.12.0 https://github.com/novnc/websockify ./opt/noVNC/utils/websockify",
+        ]
+    )
+    # Install browsers and set up repositories
+    .run_cmd(
+        [
+            "add-apt-repository ppa:mozillateam/ppa",
+            "wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -",
+            'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google-chrome.list',
+            "wget -qO- https://packages.microsoft.com/keys/microsoft.asc | apt-key add -",
+            'add-apt-repository -y "deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main"',
+            "apt-get update",
+        ],
+    )
+    # Install browsers and VS Code
+    .apt_install(["firefox-esr", "google-chrome-stable", "code"])
+    # Configure system settings
+    .run_cmd(
+        [
+            "ln -sf /usr/bin/xfce4-terminal.wrapper /etc/alternatives/x-terminal-emulator",
+            "update-alternatives --set x-www-browser /usr/bin/firefox-esr",
+            "mkdir -p /home/user/.config/Code/User",
+            "mkdir -p /home/user/.config/xfce4/xfconf/xfce-perchannel-xml/",
+            "update-desktop-database /usr/share/applications/",
+        ],
+    )
+    # Copy all configuration files
+    .copy_items(
+        [
+            CopyItem(
+                src="google-chrome.desktop",
+                dest="/usr/share/applications/google-chrome.desktop",
+            ),
+            CopyItem(
+                src="settings.json", dest="/home/user/.config/Code/User/settings.json",
+            ),
+            CopyItem(
+                src="wallpaper.png", dest="/usr/share/backgrounds/xfce/wallpaper.png",
+            ),
+            CopyItem(
+                src="xfce4-desktop.xml",
+                dest="/home/user/.config/xfce4/xfconf/xfce-perchannel-xml/xfce4-desktop.xml",
+            ),
+            CopyItem(
+                src="firefox-policies.json",
+                dest="/usr/lib/firefox-esr/distribution/policies.json",
+            ),
+            CopyItem(
+                src="firefox-autoconfig.js",
+                dest="/usr/lib/firefox-esr/defaults/pref/autoconfig.js",
+            ),
+            CopyItem(src="firefox.cfg", dest="/usr/lib/firefox-esr/firefox.cfg"),
+        ]
+    )
+)
+
+# Template with user and workdir set
+template_with_user_workdir = template.set_user("user").set_workdir("/home/user")
```

Comment:
we need this because earlier we set the user to root and workdir to /
this will set the final user and workdir to the Sandbox

Reply:
we can probably remove this and just append user=root on every command and copy (broken now)